web_calendar_color_field/readme/USAGE.rst

```diff
@@ -0,0 +1,2 @@
+Set a color attribute value on any many2one calendar field set as filter and
+set the calendar's color attribute to use the same many2one field.
```

Comment:
I don't fully understand how I should adjust the calendar view. Can you please add an example?

Reply:
Sorry about that :sweat_smile: 
Just added an example in the readme